test/name_service/utils.clj

```diff
@@ -1,2 +1,26 @@
 (ns name-service.utils
-  (:require [clojure.test :refer :all]))
+  (:require [clojure.test :refer :all]
```

Comment:
As far as I can see you are not using anything from the `clojure.test` namespace. What is the purpose of this import in this case?